islandCountTest.java

```diff
@@ -0,0 +1,66 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class islandCountTest {
+
+	// Set up of map
+	public Boolean[][] mapSetUp() {
+		
+		Boolean map[][] = new Boolean[4][4];
+		
+		map[0][0] = false;
```

Comment:
Setting up the map like this is hard to read. Why not new bool[][] {
 {W,L,W,L},
 {W,W,L,L},
  e.t.c.
};
where W and L are constants.